go-controller/hybrid-overlay/pkg/controller/node_windows.go

```diff
@@ -42,15 +42,15 @@ func newNodeController(kube kube.Interface,
 ) (nodeController, error) {
 	supportedFeatures := hcn.GetSupportedFeatures()
 	if !supportedFeatures.HostRoute {
-		return nil, fmt.Errorf("This version of windows does not support HostRoute " +
+		return nil, fmt.Errorf("this version of windows does not support HostRoute " +
```

Comment:
So this is a larger violation of the "errors should be phrase-like, not sentence-like" rule... but changing the first character to be lowercase isn't the right fix. (Likewise the next one.)

Reply:
So, I don't know what to do about that except logging it as `Klog.Error` and add a simpler message for the `fmt.Errorf` (which trickles up to: https://github.com/ovn-org/ovn-kubernetes/blob/master/go-controller/hybrid-overlay/cmd/hybrid-overlay-node/hybrid-overlay-node.go#L38). Suggestion?